script/pump.py

```diff
@@ -61,6 +61,7 @@
            | EMPTY
```

Comment:
I think we don't actually use this script any more. cc @zcbenz who I think added it originally.

Reply:
we don't: @cclauss would you please remove it instead?